src/components/LandingPages/WorkerLanding.tsx

```diff
@@ -404,6 +426,24 @@ class WorkerLanding extends Component<Props, State> {
               View Documents
             </div>
           </Link>
+
+          <Link
+            to={{
+              pathname: '/applications',
+              // State is stored in location prop
+              state: { clientUsername: client.username },
+            }}
+            className="btn btn-primary mr-2 btn-sm"
+            style={{ height: 32 }}
+          >
+
+            <div style={{ fontWeight: 'bold' }}>
```

Comment:
I think you can use the tag className="font-weight-bold" instead of inline styling.